service/app/validators/ValidationUtil.java

```diff
@@ -0,0 +1,46 @@
+package validators;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sunbird.exception.BaseException;
+import org.sunbird.exception.ValidationException;
+import org.sunbird.request.Request;
+
+public class ValidationUtil {
+
+  private static Logger logger = LoggerFactory.getLogger(ValidationUtil.class);
+
+  public static void validateRequestObject(Request request) throws BaseException {
+    if (MapUtils.isEmpty(request.getRequest())) {
+      logger.error("validateMandatoryParamsOfStringType:incorrect request provided");
+      throw new ValidationException.InvalidRequestData();
+    }
+  }
+
+  public static void validateMandatoryParamsOfStringType(
+      Request request, List<String> mandatoryParamsList) throws BaseException {
```

Comment:
List<Object> and pass another Type - String.class.
Given that you're doing this at the lowest layers, lets name this `validateMandatoryParams`